Package.swift

```diff
@@ -35,8 +35,8 @@ let package = Package(
         ),
         .binaryTarget(
             name: "SalemoveSDK",
-            url: "https://github.com/salemove/ios-bundle/releases/download/0.33.3/SalemoveSDK.xcframework.zip",
-            checksum: "6022b274530da0aa8edecbfa35492f7df2a7c4e3c6975a39b7e2b96fe96a934c"
+            url: "https://github.com/salemove/ios-bundle/releases/download/0.33.4/SalemoveSDK.xcframework.zip",
+            checksum: "${CHECKSUM}"
```

Comment:
Where does this checksum comes from?

Reply:
This happened because Xcode 13.3.x throws unnecessary xcobuildlog in response to `package compute-checksum` command. Due to this replacement has not been happened. I fixed this in our private repo and created PR.

```
[12:21:42]: $ swift package compute-checksum *redacted*/SalemoveSDK.xcframework.zip | xargs echo -n
[12:21:43]: ▸ 2022-06-14 12:21:43.636 xcodebuild[6643:34321] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionSentinelHostApplications for extension Xcode.DebuggerFoundation.AppExtensionHosts.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
[12:21:43]: ▸ 2022-06-14 12:21:43.637 xcodebuild[6643:34321] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionPointIdentifierToBundleIdentifier for extension Xcode.DebuggerFoundation.AppExtensionToBundleIdentifierMap.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
[12:21:45]: ▸ 2022-06-14 12:21:45.049 xcodebuild[6644:34332] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionSentinelHostApplications for extension Xcode.DebuggerFoundation.AppExtensionHosts.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
[12:21:45]: ▸ 2022-06-14 12:21:45.049 xcodebuild[6644:34332] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionPointIdentifierToBundleIdentifier for extension Xcode.DebuggerFoundation.AppExtensionToBundleIdentifierMap.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
[12:21:48]: ▸ 894c7b4cb4e19b6e27efbbb1b16cd94b0e205ef3f554cbb2f776f7591ccfa10e
```